ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -115,7 +115,7 @@ private void SetupNotificationSiteMap(IEnumerable<NotificationSite> notification
         }
 
         private void UpdateFlags() {
-            if (ClinicalDetails.DidNotStartTreatment) {
+            if (ClinicalDetails.DidNotStartTreatment == true) {
```

Comment:
This should just be `if (ClinicalDetails.DidNotStartTreatment)`, no?

Reply:
Correct, we don't care anything else (null or false), will change

Reply:
This one is not nullable somehow (will check if it should be)
Otherwise I get casting issue (bool? to bool) when I try to check nullable boolean without specifying exact value "false" or "true"

Reply:
If DidNotStartTreatment is nullable and we need to manually make sure the bool? - > bool conversion goes right, then a more telling way to express this would be using the ?? operator: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator
`if (ClinicalDetails.DidNotStartTreatment ?? false)`

Reply:
ah yes, it's like GetValueOrDefault